lib/src/html/template_data.dart

```diff
@@ -32,7 +32,10 @@ abstract class TemplateData<T extends Documentable> {
   String get title;
   String get layoutTitle;
   String get metaDescription;
+
   List get navLinks;
+  Documentable get navLinksLast => navLinks.last;
```

Comment:
Might we instead make name and kind getters from the templates and let the template classes define how we get them?